docs/start/getting-started/hosting.md

```diff
@@ -4,10 +4,10 @@ description: Introduction
 filterKey: integration
 ---
 
-<inline-fragment integration="react" src="~/start/getting-started/fragments/react/hosting.md"></inline-fragment>
```

Comment:
I am assuming this is correct but all web related stuff points to the react fragement.

Reply:
Yep - I moved it from React to vanillaJs because it makes more sense to have the JS content consolidated there.